gitignore

```diff
@@ -1,6 +1,7 @@
 # Logs
```

Comment:
Somehow the filename was renamed to not have the `.` prefix. It should start with a `.`.